### DISCUSSION TITLE
nuova installazione Home Assistant su Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by gericom90** [indomus/forum#771] [HUB: Home Assistant]
Buona giornata, sarei intenzionato ad installare Home Assistant su un Raspberry PI4-8GB.
Considerando che le mie conoscenze del sistema sono minime, Vi chiedo un consiglio su quale tipo di installazione fare: 
- Home Assistant OS;
- Home Assistant OS (su Docker);
- Home Assistant Core;
- Home Assistant Core (su Docker); 
o altro. 
- Il dubbio è che per semplificarmi la vita prima, non vorrei limitarmi le funzionalità future di H.A.
Un grazie anticipato per le Vostre indicazioni.

**Comment by snakuzzo** [comment]
Ciao. Leggi [QUI](https://indomus.it/focus/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/).
Nel tuo caso, visto che non ne sai quasi nulla, consiglierei HassOS. Ma non fare l'errore che fanno tutti pensando che sia un sistema "chiavi in mano". Devi comunque studiare e capire dei concetti, non pensare che sia tutto funzionante con qualche click

P.S. HassOS su Docker non esiste. Quello che intendevi è HA Supervised su Docker (su Debian) che sul tuo rpi non è il caso di fare perché appunto è ufficialmente supportato solo Debian come sistema operativo (no derivate).
